PyRDF/CallableGenerator.py

```diff
@@ -9,48 +9,6 @@ class CallableGenerator(object):
     def __init__(self, root_node):
         self.root_node = root_node
 
-    def _dfs(self, node, prev = 't', ops = []):
```

Comment:
Why is the removal of this function in this PR?

Reply:
@etejedor , this function was a part of my previous implementation of Callable Generator. I thought I could utilize this commit for some clean up as well

Reply:
Does the commit message say cleanup? :smile:

Reply:
Yup @etejedor . Fair enough. I'll separate this into another commit 👍